Gaijinizer patch/Items.txt

```diff
@@ -897,6 +904,62 @@ human princess. If you give it to a specific demon...#Triketos
 The Tengu Prince#Triketos
 > END STRING
 
+> BEGIN STRING
+崇高を意味する白色の宝石。売ってもお金になる。
+\\C[14]妖魔に対して\\C[0]プレゼントすると、友好度が上がる。
+> CONTEXT: Items/91/description
+A white jewel symbolizing nobleness. Can be sold for money.
+\\C[14]Increases friendship when given to a demon\\C[0].#Strate
+> END STRING
```

Comment:
Even though it says white in the original text, we want this to be "blue".

Reply:
fixed